kubernetes/ClusterSubscription/resource.yaml

```diff
@@ -50,6 +50,10 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+            - name: USER_AGENT_NAME
+              value: ClusterSubscription
+            - name: VERSION
+              value: {{TRAVIS_TAG}}
```

Comment:
```suggestion
            - name: USER_AGENT_NAME
              value: razee-io/clustersubscription
            - name: USER_AGENT_VERSION
              value: "{{{TRAVIS_TAG}}}"
```

Reply:
- travis needs to be in double quotes
- use `USER_AGENT_VERSION`
- just i decided i wanted to prefix the user agent name with `razee-io`, please do that for the others as well